setup.cfg

```diff
@@ -44,7 +44,7 @@ tests_require =
     webob
 
 [options.packages.find]
-exclude =
+exclude=
```

Comment:
Doesn’t seem to be needed?

Reply:
Oh Correct. I failed to revert this space-only change as I went back-and-forth a few ways of doing things.